lib/src/utils/resolveRichTextToNodes.spec.ts

```diff
@@ -83,6 +83,58 @@ describe("resolveNode", () => {
       },
       content: "I am text",
     });
+
+    // default with stringResolver
+    expect(
+      resolveNode(node, {
+        stringResolver: () => ({
+          component: "span",
+          props: { class: "this-is-text" },
+        }),
+      })
+    ).toStrictEqual({
+      content: [
+        {
+          content: "I am text",
+          component: "span",
+          props: {
+            class: "this-is-text",
+          },
+        },
+      ],
+    });
+
+    // with marks and stringResolver
+    expect(
+      resolveNode(
+        {
+          text: "I am text",
+          type: "text",
+          marks: [{ type: "bold" }],
+        },
+        {
+          stringResolver: () => ({
```

Comment:
I see in the implementation that the function supports a text prop access: `stringResolver: (text) => ({` so it might be good idea to add one more test case for affecting that. similar to what the react alternative does https://www.npmjs.com/package/storyblok-rich-text-react-renderer#text-resolver

Reply:
done